src/apps/reservation-list/modal/reservation-details/reservation-details-redirect.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC, useCallback } from "react";
+import ExternalLinkIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/buttons/icon-btn-external-link.svg";
+import { Link } from "../../../../components/atoms/link";
+import { useText } from "../../../../core/utils/text";
+import { MaterialProps } from "../../../loan-list/materials/utils/material-fetch-hoc";
+import { useDeleteV1UserReservationsIdentifier } from "../../../../core/publizon/publizon";
+import DeleteReservationModal from "../delete-reservation/delete-reservation-modal";
+import { getModalIds } from "../../../../core/utils/helpers/general";
+import { useModalButtonHandler } from "../../../../core/utils/modal";
+
+export interface ReservationDetailsRedirectProps {
+  reservationId: string;
+}
+
+const ReservationDetailsRedirect: FC<
+  ReservationDetailsRedirectProps & MaterialProps
+> = ({ reservationId }) => {
+  const t = useText();
+  const { mutate } = useDeleteV1UserReservationsIdentifier();
+  const { open, close } = useModalButtonHandler();
+  const modalIds = getModalIds();
+  const modalId = `${modalIds.deleteReservation}${reservationId}`;
+
+  const deleteReservation = useCallback(() => {
+    if (reservationId) {
+      mutate(
+        {
+          identifier: reservationId
+        },
+        {
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
+          onSuccess: (result) => {
+            close(modalId);
+          },
+          // todo error handling, missing in figma
+          onError: () => {
+            close(modalId);
+          }
+        }
+      );
+    }
+  }, [close, modalId, mutate, reservationId]);
+
+  return (
+    <>
+      <div className="modal-details__buttons">
+        <button
+          type="button"
+          onClick={() => open(modalId)}
+          className="link-tag mx-16"
+        >
+          {t("reservationDetailsRemoveReservationText")}
+        </button>
+        <Link
+          id="test-ereolen-button"
+          href={new URL("https://ereolen.dk/user/me/")}
```

Comment:
This seems like a momentary test block. I think you should clarify that with a todo comment.